notebooks/compare_esm_ssp_opt.ipynb

```diff
@@ -0,0 +1,691 @@
+{
```

Comment:
<p>It seems like this is a common function that could use a home outside of this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/ctsm_py/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Agreed!</p>